x-pack/plugins/observability_solution/inventory/public/plugin.tsx

```diff
@@ -49,6 +50,28 @@ export class InventoryPlugin
   ): InventoryPublicSetup {
     const inventoryAPIClient = createCallInventoryAPI(coreSetup);
 
+    pluginsSetup.observabilityShared.navigation.registerSections(
```

Comment:
It seems that by registering the plugin outside of the observability one, the link in the side navigation is no longer marked as active when you're on that page, which wasn’t the case before.

|Inventory|Alerts|
|-|-|
|![Screenshot 2024-09-18 at 14 51 52](https://github.com/user-attachments/assets/f9c7e4f9-b617-477c-98ca-a05d96901151)|![Screenshot 2024-09-18 at 14 52 01](https://github.com/user-attachments/assets/c9cd4534-4d02-497b-a7d7-4ddaaf127897)|

Reply:
good catch . Will fix it

Reply:
https://github.com/user-attachments/assets/de2fd99e-6773-4774-9a95-6fe6a187cb9f

Reply:
https://github.com/elastic/kibana/pull/193251/commits/7124e05836f616f64823c1a8afc675466b8f9359